### ISSUE TITLE
REQUEST - Add vSwap AMM on BSC

### ISSUE THREAD

**Issue by EteREaL85** [chartexpro/feature-requests-bugs#50]
Vswap from valuedefi is a new AMM in BSC. According to their analytics it has 140 millions in liquidity. Some tokens are currently only available in this swap and therefore they cannot be displayed in chartex, which is disappointing. Could you please include this swap in chartex? Thanks

**Comment by ivanfong01** [comment]
Agree. More and more coin place their LP there. And especially for those with non standard ratio LP
